### PR TITLE
feat: add upload content gcs endpoint

### PR DESCRIPTION
**Pull request by nnhuyhoang**
#### What's this PR do?

- [x] Add new route for uploading content : POST /api/v1/employee/:id/upload-content
- [x] Add new table content for storing media content

#### ERD
Screen Shot 2022-11-22 at 02 44 00

#### flow upload content
 validate input (ID, file) -> check file existed -> check employee existed -> create content record -> upload to GCS
 #### curl 
 ```
curl --location --request POST 'http://localhost:8200/api/v1/employees/2655832e-f009-4b73-a535-64c3a22e558f/upload-content' \
--form 'file=@"/Users/hoangnguyen/Desktop/OCPI-2.2-d2.pdf"'
 ```

### CODE REVIEW

**Comment by namnhce** [comment]
Rebase first em.

File: .env.sample [hunk 1]
```diff
@@ -11,3 +11,6 @@ DB_SSL_MODE="disable"
 ALLOWED_ORIGINS="*"
 ENV=dev
 DEBUG=true
+GOOGLE_APPLICATION_CREDENTIALS=./df-gcs.json
+GCS_BUCKET_NAME=bucketName
+GCS_PROJECT_ID=projectID
```
Comment (datphamcode295) [inline .env.sample:16]: add new line

File: pkg/config/config.go [hunk 2]
```diff
@@ -65,9 +65,11 @@ type Kafka struct {
 }
 
 type Google struct {
-	ClientSecret string
-	ClientID     string
-	AppName      string
+	ClientSecret  string
+	ClientID      string
+	AppName       string
+	GCSBucketName string
```
Comment (namnhce) [inline pkg/config/config.go:71]: How do we classify which image in the bucket used for avatar image or document files?

**Review by namnhce — CHANGES_REQUESTED** [review]

**Comment by namnhce** [comment]
rebase please

**Comment by nguyennh4522** [comment]
@nnhuyhoang a rebase + thêm env ở `service/vault/vault.go` luôn nha

**Review by datphamcode295 — APPROVED** [review]

**Review by datphamcode295 — APPROVED** [review]

**Review by datphamcode295 — APPROVED** [review]

**Review by datphamcode295 — APPROVED** [review]

**Pull request merged**